src/management/__generated/managers/ActionsManager.ts

```diff
@@ -0,0 +1,810 @@
+    if (requestParameters.id === null || requestParameters.id === undefined) {
+      throw new runtime.RequiredError(
+        'id',
+        'Required parameter requestParameters.id was null or undefined when calling delete.'
+      );
+    }
+    const queryParameters: any = {};
+
+    if (requestParameters.force !== undefined) {
+      queryParameters['force'] = requestParameters.force;
+    }
+
+    const response = await this.request(
+      {
+        path: `/actions/actions/{id}`.replace(
+          `{${'id'}}`,
+          encodeURIComponent(String(requestParameters.id))
+        ),
+        method: 'DELETE',
+        query: queryParameters,
+      },
+      initOverrides
+    );
+
+    return new runtime.VoidApiResponse(response);
+  }
+
+  /**
+   * Deletes an action and all of its associated versions. An action must be unbound from all triggers<br/>before it can be deleted.<br/>
+   * Delete an action
+   */
+  async delete(
+    requestParameters: DeleteActionRequest,
+    initOverrides?: RequestInit | runtime.InitOverrideFunction
+  ): Promise<void> {
+    await this.deleteRaw(requestParameters, initOverrides);
+  }
+
+  /**
+   * Retrieve an action by its ID.<br/>
+   * Get an action
+   * @throws {RequiredError}
+   * @memberof ActionsManager
+   */
+  async getRaw(
+    requestParameters: GetActionRequest,
+    initOverrides?: RequestInit | runtime.InitOverrideFunction
+  ): Promise<runtime.ApiResponse<GetActions200ResponseActionsInner>> {
+    if (requestParameters.id === null || requestParameters.id === undefined) {
+      throw new runtime.RequiredError(
+        'id',
+        'Required parameter requestParameters.id was null or undefined when calling get.'
+      );
+    }
+
+    const response = await this.request(
+      {
+        path: `/actions/actions/{id}`.replace(
+          `{${'id'}}`,
+          encodeURIComponent(String(requestParameters.id))
+        ),
+        method: 'GET',
+      },
+      initOverrides
+    );
+
+    return new runtime.JSONApiResponse(response);
+  }
+
+  /**
+   * Retrieve an action by its ID.<br/>
+   * Get an action
+   */
+  async get(
+    requestParameters: GetActionRequest,
+    initOverrides?: RequestInit | runtime.InitOverrideFunction
+  ): Promise<GetActions200ResponseActionsInner> {
+    const response = await this.getRaw(requestParameters, initOverrides);
+    return await response.value();
+  }
+
+  /**
+   * Retrieve a specific version of an action. An action version is created whenever<br/>an action is deployed. An action version is immutable, once created.<br/>
+   * Get a specific version of an action
+   * @throws {RequiredError}
+   * @memberof ActionsManager
+   */
+  async getVersionRaw(
+    requestParameters: GetActionVersionRequest,
+    initOverrides?: RequestInit | runtime.InitOverrideFunction
+  ): Promise<runtime.ApiResponse<GetActionVersions200ResponseVersionsInner>> {
+    if (requestParameters.actionId === null || requestParameters.actionId === undefined) {
```

Comment:
We can just do `if (requestParameters.actionId == null) {` - (will be better for test coverage too)

https://github.com/lodash/lodash/blob/master/isNil.js